source/connect/ssh-connection.txt

```diff
@@ -11,3 +11,156 @@ Proxy / SSH Tunnel Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`Proxy / SSH Tunnel` tab allows you to connect to your deployment using 
+a Proxy method or SSH Tunnel. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Proxy / SSH Tunnel` tab.
+
+      You have the following connection options:
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50 
+      
+         * - Option
+           - Description 
+
+         * - SSH with Password 
+           - Connects to a deployment using a SSH Tunnel and has the option to secure 
+             the connection with a password. For information on SSH connections and 
+             the fields for this connection option, see :ref:`<ssh-fields>`.
+
+         * - SSH with Identity File 
+           - Connects to a deployment using a SSH Tunnel and uses an Identity File 
+             to secure the connection. For information on SSH connections and 
+             the fields for this connection option, see :ref:`<ssh-fields>`.
+
+         * - Socks5
+           - Connects to a deployment using a seperate proxy server. For more 
+             information on the fields for this connection option, see 
+             :ref:`<socks5>`. 
+
+      .. _ssh-fields:
+ 
+      SSH Connection
+      ~~~~~~~~~~~~~~
+
+      Use the following fields to connect to a deployment using 
```

Comment:
"To connect to ...., use ..." ? Depends on how the rest of the page is written though.

Reply:
Done